core/embed/rust/src/ui/model_tt/component/page.rs

```diff
@@ -495,8 +494,10 @@ where
 
 #[cfg(test)]
 mod tests {
+    extern crate serde_json;
```

Comment:
Switched to `use` instead of `extern create` in [b3bf5f0](https://github.com/trezor/trezor-firmware/pull/2803/commits/b3bf5f00d6d3ee4ada60ea48769566ea17d61ba1)